scripts/azure-const.sh

```diff
@@ -1,31 +1,31 @@
 #!/bin/bash
-if [ -z "$CLIENT_ID" ]; then
+if [ -z "$AZURE_CLIENT_ID" ]; then
```

Comment:
what's this script used for? is it going to break any jobs that rely on it to change the env vars?

Reply:
This script is used to generate `azureconst.go`, it won't break anything I am just doing this so that we remain consistent across acs-engine. Packer uses env vars prefixed with `AZURE_` for instance